addons/hr_timesheet/models/hr_timesheet.py

```diff
@@ -48,11 +48,20 @@ def _domain_task_id(self):
     employee_id = fields.Many2one('hr.employee', "Employee", domain=_domain_employee_id)
     department_id = fields.Many2one('hr.department', "Department", compute='_compute_department_id', store=True, compute_sudo=True)
     encoding_uom_id = fields.Many2one('uom.uom', compute='_compute_encoding_uom_id')
+    is_overtime = fields.Boolean('Is Overtime')
+    overtime_amount = fields.Monetary("Overtime Cost", default=0.0)
+    overtime_duration = fields.Float("Overtime Duration", default=0.0)
+    total_amount = fields.Monetary('Total Amount', compute='_compute_total_amount', store=True)
 
     def _compute_encoding_uom_id(self):
         for analytic_line in self:
             analytic_line.encoding_uom_id = analytic_line.company_id.timesheet_encode_uom_id
 
+    @api.depends('unit_amount')
```

Comment:
The triggers seem incorrect.
```suggestion
    @api.depends('overtime_amount', 'amount')
```